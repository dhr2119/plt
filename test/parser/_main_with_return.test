INT MAIN LPAREN RPAREN LBRACE RETURN SEMI RBRACE
