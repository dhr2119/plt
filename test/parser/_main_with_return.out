ACCEPT
[program:
  [mdecl:
    INT
    MAIN
    LPAREN
    RPAREN
    LBRACE
    [vdecl_list:]
    [stmt_list: [stmt_list:] [stmt: RETURN SEMI]]
    RBRACE
  ]
  [fdecls:]
  EOF
]
